src/ml_tooling/storage/file.py

```diff
@@ -0,0 +1,106 @@
+from ml_tooling.storage.base import Storage, StorageEnvironment
+from ml_tooling.utils import MLToolingError
+
+from typing import List, Any
+from contextlib import contextmanager
+from pathlib import Path
+from joblib import load, dump
+
+
+class FileStorage(Storage):
+    
+    def __init__(self, dir_path=None):
+        self.dir_path = Path.cwd() if dir_path is None else Path(dir_path)
+        self.filename = None
+        self.file = None
+
+    def __enter__(self):
+        if not self.dir_path.exists():
+            self.dir_path.mkdir(parents=True)
+        return self
+
+    def __exit__(self, exc_type, exc_value, traceback):
+        return False
+
+    def get_list(self) -> List[Path]:
+        """
+        Finds a list of estimator filenames in the FileStorage directory,
+        if the path given is to a file, the directory in which the file resides
+        is used to find the list.
+
+        Parameters
+        ----------
+        None
+
+        Example
+        -------
+        Find and return estimator paths in a given directory:
+            my_estimators = FileStorage('path/to/dir').get_list()
+
+        Returns
+        -------
+        List[Path]
+            list of paths to files
+        """
+        return sorted(self.dir_path.glob('*.pkl'))
+    
+    def load(self, file_path) -> Any:
```

Comment:
Returns an Estimator as it unpickles in the end

Reply:
but any object can be pickled :)

Reply:
We can typehint `file_path` here